src/schematics/ng-add/index_spec.ts

```diff
@@ -73,3 +83,23 @@ describe('ng-add', () => {
     expect((npmInstallTask.options as any).command).toBe('install');
   });
 });
+
+
+describe('ng-add-angular8-specific', () => {
+  let runner: SchematicTestRunner;
+  let appTree: UnitTestTree;
+
+  const defaultOptions: NgAddOptions = {};
+
+  beforeEach(() => {
+    appTree = createTestApp();
+    createCommonWebpackConfigForAngular8(appTree);
+    runner = new SchematicTestRunner('schematics', path.join(__dirname, '../collection.json'));
+  });
+
+  it('sthe typescript.js file in webpack config should have directTemplateLoading: false', () => {
```

Comment:
`sthe typescript.js` => `the typescript.js`

Reply:
Oops... sorry about that.